assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -385,10 +396,21 @@ def finetune(args: Namespace):
     logger.info(f"Precision: {getattr(args, 'precision', None)}")
 
     # set `ignore_mismatched_sizes` to `false` by default
-    if hasattr(args, "model_name") and args.model_name in IGNORE_MISMATCHED_SIZES_FALSE_MODELS:
-        logger.info(f"Forcing `ignore_mismatched_sizes` to False for {args.model_name}")
+    if hasattr(args, "model_type") and args.model_type in IGNORE_MISMATCHED_SIZES_FALSE_MODELS:
+        logger.info(f"Identified model type: {args.model_type}. Forcing `ignore_mismatched_sizes` to False.")
         setattr(args, "ignore_mismatched_sizes", False)
 
+    # set `mlflow_ft_conf` - contains all mlflow related properties
+    setattr(args, "mlflow_ft_conf", {})
+
+    # set task based mlflow_model_signature
+    if getattr(args, "task_name", None) is not None and args.task_name in MLFLOW_MODEL_SIGNATURES:
```

Comment:
Are we adding signatures? We will not be able to the schema with parameters for online deployment then